amplipi/streams.py

```diff
@@ -284,6 +284,7 @@ def disconnect(self):
       try:
         # must use terminate as kill() cannot be intercepted
         self._cproc.terminate()
+        self._cproc.communicate(timeout=5)
```

Comment:
`terminate()` just fires off a signal; `communicate` actually prevents zombies from piling up. Doing things this way can also more appropriately bubble up errors caught during a timeout exception. This was ultimately not the cause of the issue tho, just a worthy improvement.